src/pageLayout/containers/TreeNav.tsx

```diff
@@ -39,74 +39,149 @@ const TreeSidebar: FC = () => {
     }
   }
 
+  useEffect(() => {
+    const helpBarMenu = document.querySelectorAll<HTMLElement>(
```

Comment:
Ah this does seem a bit hacky, but no way around it. The toggleNavExpansion doesn't take an element, otherwise we could add this logic in the above section. Anyways looks good!